### PR TITLE
logs print out to files

### PR DESCRIPTION
**Pull request by amit-levints-bgu**

### CODE REVIEW

File: src/main/java/LoggingExample.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import java.util.logging.Logger;
```
Comment (inbarbc) [inline src/main/java/LoggingExample.java:1]: still need this file?

File: src/main/java/Server/ShopController.java [hunk 2]
```diff
@@ -8,11 +8,11 @@
 
 @RestController
 @RequestMapping(path = "/api/user")
-public class ShopAPI {
+public class ShopController {
```
Comment (inbarbc) [inline src/main/java/Server/ShopController.java:11]: thanks

File: src/main/java/Server/ShopController.java [hunk 3]
```diff
@@ -8,11 +8,11 @@
...
     private final ShopService _shopService;
 
     @Autowired
-    public ShopAPI(ShopService shopService) {
+    public ShopController(ShopService shopService) {
```
Comment (inbarbc) [inline src/main/java/Server/ShopController.java:15]: thanks

File: src/main/java/Server/SystemController.java [hunk 4]
```diff
@@ -10,11 +10,11 @@
 
 @RestController
 @RequestMapping(path = "/api/system")
-public class SystemAPI {
+public class SystemController {
```
Comment (inbarbc) [inline src/main/java/Server/SystemController.java:13]: thanks

File: src/main/java/Server/SystemController.java [hunk 5]
```diff
@@ -10,11 +10,11 @@
...
     private final SystemService _systemService;
 
     @Autowired
-    public SystemAPI(SystemService systemService) {
+    public SystemController(SystemService systemService) {
```
Comment (inbarbc) [inline src/main/java/Server/SystemController.java:17]: thanks

File: src/main/resources/logging.properties [hunk 6]
```diff
@@ -0,0 +1,13 @@
+handlers=java.util.logging.FileHandler, java.util.logging.ConsoleHandler
```
Comment (inbarbc) [inline src/main/resources/logging.properties:1]: those are the limits on the logs files?

File: src/main/java/LoggingExample.java (hunk 1 above)
Comment (amit-levints-bgu) [reply src/main/java/LoggingExample.java:1]: Yes, we will need this to copy from it. I will delete it in the next two weeks :)

File: src/main/resources/logging.properties (hunk 6 above)
Comment (amit-levints-bgu) [reply src/main/resources/logging.properties:1]: yes. we can easily change it if needed

File: src/main/java/LoggingExample.java (hunk 1 above)
Comment (inbarbc) [reply src/main/java/LoggingExample.java:1]: trust you. dont forget

File: src/main/resources/logging.properties (hunk 6 above)
Comment (inbarbc) [reply src/main/resources/logging.properties:1]: perfect thanks

**Review by inbarbc — APPROVED** [review]

**Pull request merged**